### ISSUE TITLE
Directory structure

### ISSUE THREAD

**Issue by moodmosaic** [hedgehogqa/dotnet-hedgehog#76]
We currently have it setup to look like this:

```
.
+-- ...
+-- Hedgehog
+-- Hedgehog.Tests
+-- ...
```

Although, the most projects I see on GitHub use this structure:

```
.
+-- ...
+-- src/Hedgehog
+-- tests/Hedgehog.Tests
+-- ...
```

If you also agree, I will open a pull request for it.

**Comment by jystic** [comment]
Is that common for .NET projects? That's the structure I use for Haskell things, but cabal works differently to .NET

From my experience in .NET land you normally have a directory per-assembly, but things may have changed.

**Comment by moodmosaic** [comment]
Yeah, it looks common now. See [here](https://gist.github.com/davidfowl/ed7564297c61fe9ab814) for example. Also, some random projects I've looked at use the same structure ([link](https://github.com/Microsoft/visualfsharp), [link](https://github.com/eiriktsarpalis/TypeShape), [link](https://github.com/xyncro/chiron)).

**Comment by moodmosaic** [comment]
Resolved by #78.
